src/sentry/deletions/defaults/group.py

```diff
@@ -106,6 +106,7 @@ def get_unfetched_events(self) -> list[Event]:
                 ]
             )
 
+        logger.info("Fetching %s events for deletion.", self.DEFAULT_CHUNK_SIZE)
```

Comment:
I don't need this forever. I can delete it after we get to the bottom of my investigation.